met/src/libcode/vx_statistics/met_stats.cc

```diff
@@ -1569,6 +1570,12 @@ void VL1L2Info::calc_ncep_stats() {
 
    DIR_ABSERR   = fabs(DIR_ERR);
 
+   char buffer[1024];
```

Comment:
@hsoh-u this looks like temporary code that you added to debug this issue. But we wouldn't want to actually include this in the release. Recommend removing.